src/taipy/gui/extension/library.py

```diff
@@ -87,6 +88,8 @@ class Element:
     what the default property name is.
     """
 
+    __RE_PROP_VAR = re.compile(r"<tp:prop:(\w+)>")
```

Comment:
Would this work if the var name is a property in a dict, if this makes sense?
like `my_dict.my_entry`, as '.' is not captured.

Reading the code... it seems you're locating an element property name, that cannot contain dots, so we're good.